cloud/endagaweb/models.py

```diff
@@ -58,6 +58,9 @@
 OUTBOUND_ACTIVITIES = (
     'outside_call', 'outside_sms', 'local_call', 'local_sms',
 )
+# These UsageEvent events are not allowed block the Subscriber if repeated
+# for 3 times
+INVALID_EVENTS = ('error_call', 'error_sms')
```

Comment:
We should keep this list empty/commented out until we define the use cases and implement/validate the dialplan/chatplan event generation